platforms/chibios/drivers/serial.c

```diff
@@ -18,9 +18,6 @@
 //   this method seems to be more accurate than GPT timers
 #if PORT_SUPPORTS_RT == FALSE
 #    error "chSysPolledDelayX method not supported on this platform"
-#else
-#    undef wait_us
-#    define wait_us(x) chSysPolledDelayX(US2RTC(CPU_CLOCK, x))
```

Comment:
Can we put this back for now so that it doesn't accidentally use `WAIT_US_TIMER`

Reply:
Yes, will do that!

Reply:
Done.